.github/actions/unit-tests/action.yml

```diff
@@ -0,0 +1,18 @@
+name: 'Unit Tests'
+description: 'Run unit tests using go'
+inputs:
+  GO_VERSION:
+    default: "1.16"
+env:
+  GO111MODULE: "on"
+  GOPROXY: "https://proxy.golang.org"
+runs:
+  using: "composite"
+  steps: 
+    - name: Install Go
+      uses: actions/setup-go@v2.1.4
+      with:
+        go-version: ${{ inputs.GO_VERSION }}
+    - name: Test
+      shell: bash
+      run: go test -coverprofile=coverage.txt -covermode=atomic -v ./...
```

Comment:
It doesn't seem that you're making use of the coverage file after generating it. I would suggest to either make use of it in some way or to remove the coverage generation.

Reply:
fixed